lib/omniauth/strategies/vitae.rb

```diff
@@ -29,9 +29,10 @@ class Vitae < OmniAuth::Strategies::OAuth2
       def raw_info
         @raw_info ||= begin
           modify_connection
-          access_token.get('/api/users/self',
-                           :headers => { 'Accept' => 'application/json' }).
-            parsed.fetch('results').first
+          access_token.get('/api/users/self', :headers => {
```

Comment:
I think `:headers` should be on it's own line, then the following lines 33-34 spaced in.